netbuf/netbuf_write.c

```diff
@@ -230,6 +230,7 @@ uint8_t *
 netbuf_write_reserve(struct netbuf_write * W, size_t len)
 {
 	struct writebuf * WB;
+	struct writebuf * OB;		/* Old buffer. */
```

Comment:
Why not just reuse `WB` for this?

Reply:
I figured that

>    "use `struct writebuf * OB` instead of `W->tail`"

would be a better git commit message than

>  "use `struct writebuf * WB` to mean `W->tail` sometimes, in addition to also being a new malloced buffer in the same function".

No, really -- the first version of this commit used WB for everything, but when I was double-checking the branch history, it looked like yuck.

Granted, this is a concern for `netbuf_write_reserve` and not `netbuf_write_consume`, but once I'd introduced OB for `_reserve`, I figured that it made sense to use the same name for `_consume`.

... I mean, ok, I'll switch them both to WB if that's definitely what you want.

Reply:
"Refactor code to extract W->tail into a variable for reuse", perhaps?

Reply:
I just went with "use `struct writebuf * WB` instead of W->tail".  Evidently I was over-thinking it.